package.json

```diff
@@ -73,6 +74,8 @@
     "meow": "^3.3.0",
     "object-assign": "^4.0.1",
     "plur": "^2.0.0",
+    "power-assert": "^1.0.0",
+    "power-assert-renderers": "twada/power-assert-renderers",
```

Comment:
Just released [power-assert-renderers 0.1.0](https://www.npmjs.com/package/power-assert-renderers)

@uiureo Would you update `package.json`?